문자열-다루기-기본1.js

```diff
@@ -0,0 +1,9 @@
+function solution(s) {
+	let num = s.length
+
+	if(num === 4 || num === 6){
+		return s.split("").every((c)=>!isNaN(c))
+	}
+
+	return false;
+}
```

Comment:
오 every 메서드 잘하셨어요 ㅎㅎ
이제 컨벤션만 맞춘다면 훨씬 좋아지실 것 같아요

- 변하지 않는 값은 const로 선언해 주세요
- if문으로 return 해줬으니 else로 처리해주는게 좋을 것 같아요
```suggestion
function solution(s) {
	const num = s.length

	if (num === 4 || num === 6) {
		return s.split("").every((c)=>!isNaN(c))
	} else return false;
}
```